Assets/MixedRealityToolkit/Attributes/MixedRealityControllerAttribute.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
 using System;
 using System.Linq;
+using System.Reflection;
```

Comment:
Is this only used for editor code? If so, please consider wrapping in an editor preprocessor check

Reply:
Added to the .NET backend ifdef down below.